src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/header.tsx

```diff
@@ -12,10 +12,10 @@ const Header = styled('div')`
 
   background: #fff;
   font-size: ${p => p.theme.fontSizeExtraLarge};
-  @media (min-width: ${props => props.theme.breakpoints[0]}) {
+  @media (max-width: ${props => props.theme.breakpoints[1]}) {
     margin-top: 54px;
   }
-  @media (min-width: ${props => props.theme.breakpoints[1]}) {
+  @media (max-width: calc(${props => props.theme.breakpoints[0]} - 1px)) {
```

Comment:
@EvanPurkhiser should i just hardcode this instead of calc?

Reply:
The calc is probably okay since it's doing minimal computation. I doubt this has a performance impact